sdks/python/apache_beam/yaml/standard_io.yaml

```diff
@@ -53,6 +53,8 @@
     # 'WriteToBigQuery': 'apache_beam.yaml.yaml_io.write_to_bigquery'
     'ReadFromText': 'apache_beam.yaml.yaml_io.read_from_text'
     'WriteToText': 'apache_beam.yaml.yaml_io.write_to_text'
+    'ReadFromPubSub': 'apache_beam.yaml.yaml_io.read_from_pubsub'
```

Comment:
`apache_beam.yaml.yaml_io` is a little redundant. was `apache_beam.yaml.io` considered before?